gobblin-core/src/test/java/gobblin/source/extractor/filebased/TokenizedFileDownloaderTest.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (C) 2014-2016 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed
+ * under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package gobblin.source.extractor.filebased;
+
+import java.io.ByteArrayInputStream;
+import java.io.InputStream;
+import java.io.UnsupportedEncodingException;
+
+import org.testng.annotations.Test;
+
+import avro.shaded.com.google.common.base.Joiner;
+import junit.framework.Assert;
```

Comment:
The above two imports are incorrect, should use `testng` assert, and non-shaded guava